cluster-autoscaler/cloudprovider/azure/azure_scale_set_test.go

```diff
@@ -43,6 +44,32 @@ func newTestScaleSet(manager *AzureManager, name string) *ScaleSet {
 	}
 }
 
+func newTestVMSSList() []compute.VirtualMachineScaleSet {
```

Comment:
Can we accept the "capacity, name, location" as parameters? It's easier to see it from the unit test package when doing the validation.